docs/network-policy.md

```diff
@@ -157,5 +217,5 @@ ephemeral and unpredictable.
 ## Notes
 
 - The v1alpha1 CNP CRD supports up to 10000 unique priority at policy level. In
```

Comment:
We can support more than 10,000 unique priorities for each tier now, even without single tier per table. However I think it is fine to keep it 10,000 since this should be large enough for v0.9 release. Another nit is that with the introduction of tiers, we need to define more carefully what `unique priority at policy level` means.

Reply:
the 10k limit is currently enforced in yaml validation so the doc is reflecting that.. we can update limits and have the doc reflect the same later
updated the example of rule enforcement.. to include tier

Reply:
Thanks for the explanation